src/conventional/balloon_sonde_json2ioda.py

```diff
@@ -0,0 +1,310 @@
+#!/usr/bin/env python3
+
+#
+# (C) Copyright 2020-2023 UCAR
+#
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+#
+
+
+"""
+Python code to ingest JSON WindBorne Data
+"""
+
+import pdb
```

Comment:
```suggestion
```

usually do the import of `pdb` on demand leave it as bare as possible in pre-loaded libraries